lib/model/CProbabilityAndInfluenceCalculator.cc

```diff
@@ -370,8 +416,11 @@ void doComputeInfluences(model_t::EFeature feature,
     for (auto i = influencerValues.begin(); i != influencerValues.end(); ++i) {
         params.weights(weights).updateAnomalyModel(false);
 
-        computeInfluencedValue(value, count, i->second.first, i->second.second,
-                               params, influencedValue[0]);
+        if (computeInfluencedValue(value, count, i->second.first, i->second.second,
+                                   params, influencedValue[0]) == false) {
+            LOG_ERROR(<< "Failed to compute influencer value");
```

Comment:
@tveasey do you have a better idea for this error message?

Reply:
I think, at least for us it is very useful to see the function arguments, i.e. `value`, `count`, `i->second.first` and `i->second.second`.